### ISSUE TITLE
Bug: Cannot add new IMAP account

### ISSUE THREAD

**Issue by alexpdr** [Foundry376/Mailspring#1856]
**Describe the bug**
Program errors out when attempting to add a imap account.

**Trace**
```
user@computer:~$ mailspring --verbose
Gtk-Message: 14:51:01.004: Failed to load module "appmenu-gtk-module"
Running database migrations
App load time: 139ms

{"error":null}
Manual update check (updates.getmailspring.com/check/linux/x64/1.7.4-d54ff730/82ff93b8-28fa-4cd8-b810-8d19fc431ae2/stable) returned 204
{ TypeError: Cannot read property 'emailAddress' of undefined
    at _AccountStore.addAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/src/flux/stores/account-store.ts:223:18)
    at OnboardingStore._onFinishAndAddAccount (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/onboarding-store.ts:129:20)
    at EventEmitter.eventHandler (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:36:22)
    at EventEmitter.emit (/usr/share/mailspring/resources/app.asar/node_modules/eventemitter3/index.js:72:35)
    at Function.trigger (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/PublisherMethods.js:52:26)
    at Object.functor [as finishAndAddAccount] (/usr/share/mailspring/resources/app.asar/node_modules/reflux/src/createAction.js:32:56)
    at onboarding_helpers_1.finalizeAndValidateAccount.then.validated (file:///tmp/nylas-build/electron-packager/linux-x64/mailspring-linux-x64/resources/app/internal_packages/onboarding/lib/decorators/create-page-for-form.tsx:170:29)
  message: 'Cannot read property \'emailAddress\' of undefined' } { pluginIds: [ 'onboarding' ] }
```

**To Reproduce**
1. Go to "Add Account"
2. Click on "IMAP/SMTP"
3. Click through menu (adding your account /w credentials)
4. See trace (No new account gets added)

**Expected behavior**
The email account gets added to the client and shows up in the most-left sidebar. 

**Setup: (please complete the following information):**
 - OS: Pop!_OS (Ubuntu derivative)
 - OS Version: 19.10
 - Package method: .deb
 - Mailspring Version: ex: 1.7.4-d54ff730

**Additional context** 
SMTP host smtp.transip.email
IMAP host: imap.transip.email

**Comment by alexpdr** [comment]
`message: 'Cannot read property \'emailAddress\' of undefined'` Kind of makes it seem like the input validation has gone wrong in a newer release? Bug started after updating, though I can't say what the old version I had was.

**Comment by sarovin** [comment]
I think that is a duplicate #1841

**Comment by alexpdr** [comment]
@sarovin Not exact same, I get no white screen for me it just disappears. But the root cause might be the same?

**Comment by sarovin** [comment]
> @sarovin Not exact same, I get no white screen for me it just disappears. But the root cause might be the same?

Maybe

**Comment by Miyazono1** [comment]
i have a similar issue. it says the account was added like normal, but it isn't actually added. i tried doing it multiple times and restarting the application, but it still does the same thing. says account was added, but it was not actually added.

**Comment by mdzierzecki** [comment]
I have the same problem. Installing previous version (1.7.2) allows to skip this issue

**Comment by GitTiago** [comment]
Reporting the same issue. Office365 account couldn't be added on 1.7.4. @mdzierzecki 's suggestion to use 1.7.2 for now worked.

**Comment by PrijaveRazne** [comment]
IMAP account not added in 1.7.4. 
Would be nice to get a fix soon

**Comment by pythonInRelay** [comment]
Temp solution. Install an older version located [here](https://github.com/Foundry376/Mailspring/releases/tag/1.7.2).

**Comment by alexpdr** [comment]
@pythonInRelay thanks for the link!

**Comment by working-name** [comment]
I'm experiencing the same issue as @Miyazono1  with 1.7.4, 1.7.2, 1.7.1 ... I'll keep going down in versions hopefully one of them will work :)

**Comment by mensur** [comment]
I have exactly the same bug... 
I have added new account, it says that the account is added, but nothing shows in Mailspring.. 
Version: 1.7.4-d54ff730, Windows 10

**Comment by CarlosRebolledoA** [comment]
Mismo error, al agregar cuenta Outlook sale en blanco. Es un asunto importante y prioritario a resolver y no se ven respuestas de Ben

Windows 10
Versión 1.7.4.

**Comment by bengotow** [comment]
Hey folks—thanks for filing this, this is a pretty serious bug. It was reported in a handful of other issues and we shipped a fix for it yesterday in version 1.7.5. 

You can download the update from GitHub releases or from https://getmailspring.com/download, and the app should also auto-update if you have been using it. Sorry for the delay getting a fix out for this one, I appreciate everyone's help debugging and triaging the problem!
